claims-related/scripts/single_source_conversion_helpers.py

```diff
@@ -0,0 +1,334 @@
+"""Methods to assist in the conversion of single source Excel to JSON."""
+import collections
+import re
+
+import numpy as np
+import pandas as pd
+
+DX_CODE_CATEGORY = ['DX_CODE']
+DX_CODE_X_CATEGORY = ['DX_CODE_X']
+ENC_PROC_CODE_CATEGORY = ['ENCOUNTER_CODE', 'PROC_CODE']
+QUALITY_CODE_CATEGORY = ['PD_Exe', 'PD_Exl', 'PN_X', 'PN']
+
+# Descriptions of these options can be found at
+# https://cmsgov.github.io/qpp-submissions-docs/schemas# under "Single-Performance Rates".
+PERFORMANCE_OPTIONS = {
+    'PN_X': 'performanceNotMet',
+    'PN': 'performanceMet',
+    'PD_Exe': 'performanceException',
```

Comment:
These fields should map to `eligiblePopulationException` and `eligiblePopulationExclusion` instead.